diff --git a/crates/atuin/src/command/client/search/history_list.rs b/crates/atuin/src/command/client/search/history_list.rs
index bed883c7..2297769d 100644
--- a/crates/atuin/src/command/client/search/history_list.rs
+++ b/crates/atuin/src/command/client/search/history_list.rs
@@ -226,7 +226,7 @@ impl DrawState<'_> {
         {
             row_highlighted = true;
             // if not applying alternative highlighting to the whole row, color the command
-            style = self.theme.as_style(Meaning::AlertError);
+            style = self.theme.as_style(Meaning::Annotation);
             style.attributes.set(style::Attribute::Bold);
         }
 
